packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -457,6 +472,90 @@ export function RecordTable_Sequences(
       />
     ) : null;
 
+  const resetProteinFilterButton = (
+    <OutlinedButton
+      text="Reset protein filter"
+      onPress={() => {
+        setProteinFilterIds([]);
+      }}
+    />
+  );
+
+  const updateProteinFilterIds = () => {
+    setProteinFilterIds(highlightedNodes);
+    setHighlightedNodes([]);
+  };
+
+  const proteinFilter = (
+    <PopoverButton
+      buttonDisplayContent={`Proteins${
+        volatileProteinFilterIds.length > 0
+          ? ` (${volatileProteinFilterIds.length})`
+          : ''
+      }`}
```

Comment:
Should we change the label when rows are selected, to indicate that the underlying filter is operable?

Reply:
I've added an asterisk just as a placeholder (though the asterisk is also used inside the popover, so maybe it's good)